modules/data.atmosphere/R/download.AmerifluxLBL.R

```diff
@@ -19,10 +19,13 @@
 ##' 
 ##' @author Ankur Desai, based on download.Ameriflux.R by Josh Mantooth, Rob Kooper
 download.AmerifluxLBL <- function(sitename, outfolder, start_date, end_date, 
-                                  overwrite = FALSE, verbose = FALSE, username = "pecan", ...) {
+                                  overwrite = FALSE, verbose = FALSE, username = "pecan", method=NULL, ...) {
   # get start/end year code works on whole years only
   
-  
+  # set hostname to select appropriate download
+  #if (is.null(host)) host <- system("hostname", intern=T)
+  if (system("hostname", intern=T)=="modex") method <- "ncftpget"  
```

Comment:
Not a good idea to bury machine-specific hacks into low-level functions. If method is an argument to this function, why not do this check outside the function and pass the correct value of method in? Otherwise we don't have a system that works the next time someone is behind a firewall